packages/rum/test/utils.ts

```diff
@@ -1,4 +1,13 @@
-import { DeflateWorker, DeflateWorkerAction, DeflateWorkerListener } from '../src/domain/segmentCollection'
+import {
+  DeflateWorker,
+  DeflateWorkerAction,
+  DeflateWorkerListener,
+  // To avoid importing too much code in E2E tests, import from deflateWorker directly. This solves
+  // the "Timeout" typing conflict between DOM and @types/node for now. We might find a better
+  // solution in the future.
+  // (cf https://github.com/Microsoft/TypeScript/issues/30128)
+  // eslint-disable-next-line local-rules/disallow-protected-directory-import
```

Comment:
Did you consider prefixing `setTimeout` with `window` to disambiguate the conflict?

Reply:
I didn't went with it because this is what we wanted to avoid by doing #708. Another solution would be to use something like:

```ts
type Timeout = ReturnType<typeof setTimeout>
```

and use it instead of `number`. WDYT?

Reply:
That seems nicer to me 👍